.github/workflows/linuxPullRequest.yml

```diff
@@ -0,0 +1,28 @@
+name: CI Linux PR
+on:
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
A scheduled trigger is not needed for the pull requests. You only need this to test the main branch